### DISCUSSION TITLE
vscode intellisense not working for chaincode development

### DISCUSSION THREAD

**Discussion by sauravsavarn** [microsoft/vscode-discussions#695] [Extension Development QnA] [answered]
VScode is not supporting code completion (Intellisense support) for Hyperledger Fabric chaincode development.The language used to develop chaincode is NodeJS

**Comment by burkeholland** [comment]
I think you're asking for a chiancode extension here. Have you tried https://marketplace.visualstudio.com/items?itemName=Spydra.hyperledger-fabric-debugger

**Reply by sauravsavarn** [reply]
yes, but still the code suggestions for the library already included as part of package.json is not visible on press ion ctrl+space.

**Reply by gjsjohnmurray** [reply, marked as answer]
I suggest you raise that with the extension author, either by picking the extension when using VS Code's Report Issue option from the Help menu or by going directly to https://github.com/spydra-tech/fabric-debugger/issues.
